vda5050_connector/vda5050_connector_py/vda5050_controller.py

```diff
@@ -1171,8 +1171,8 @@ def _accept_order(self, order: VDAOrder, mode: OrderAcceptModes):
         # Only on stitching updates the node and edges base states are kept
         self._update_state(
             {
-                "order_id": order.order_id,
-                "order_update_id": order.order_update_id,
+                "orderId": order.order_id,
+                "orderUpdateId": order.order_update_id,
```

Comment:
This are already handled by `vda5050_connector_py/mqtt_bridge.py` using `convert_ros_message_to_json`, aren't they ?